### PR TITLE
Bugfix/net2 ntp

### PR DESCRIPTION
**Pull request by k8soneill**
Allows comma delimited list of ntp sources for net2 nodes

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# This is a simple change.
```
Comment (stevemul) [inline deploy.sh]: Err? Probably shouldn't be here - have you rolled in some test changes from when you're git pushes weren't working?

File: files/setup_bastion.yaml [hunk 2]
```diff
@@ -34,6 +34,7 @@
     setNodeRoutes: __set_node_routes__
     nodeRoutes: __node_routes__
     bastionIp: __bastion_ip__
+    net2NTPServer: __net2_ntp_server__
```
Comment (stevemul) [inline files/setup_bastion.yaml]: Shouldn't this be plural to reflect that we expect a list?

File: files/setup_bastion.yaml [hunk 3]
```diff
@@ -228,6 +229,7 @@
           bastion_ip: {{ bastionIp }}
           {% if multinetwork %}
           net2_routes: {{ net2Routes | to_json }}
+          net2_ntp_server: {{ net2NTPServer }}
```
Comment (stevemul) [inline files/setup_bastion.yaml]: servers... as above

**Review by stevemul — DISMISSED** [review]
Some cleanup required. Also has it been tested?

**Review by MudasarUKCloud — APPROVED** [review]
looks fine to me

**Comment by MudasarUKCloud** [comment]
looks fine to me

**Review by stevemul — APPROVED** [review]
lgtm

**Pull request merged**